core/deployment/src/main/java/io/quarkus/deployment/dev/DevConfig.java

```diff
@@ -10,7 +10,8 @@
     /**
      * Whether or not Quarkus should disable it's ability to not do a full restart
      * when changes to classes are compatible with JVM instrumentation.
-     * If this is set to true, Quarkus will always restart on changes and never perform class redefinition.
+     * 
+     * If this is set to false, Quarkus will always restart on changes and never perform class redefinition.
      */
     @ConfigItem(defaultValue = "true")
     boolean instrumentation;
```

Comment:
Why do we have a `DevConfig` while we already have a `LiveReloadConfig`?

Reply:
LiveReload config is basically for remote-dev.

Reply:
Perhaps it should be renamed...

Reply:
Well... we only had live reload config for remote dev until now. But I think `quarkus.live-reload.instrumentation` is a much better name than `quarkus.dev.instrumentation` and would be more consistent with the rest.

I can do the legwork if you all agree with it.

Reply:
Fine with me :)

Reply:
Yea, in hindsight it should have been under LiveReload, but because it was just for remote dev so far I did not think of it.